doc/nrf/protocols/wifi/promiscuous_operation.rst

```diff
@@ -0,0 +1,155 @@
+.. _ug_nrf70_developing_promiscuous_packet_reception:
+
+Raw IEEE 802.11 promiscuous packet reception
```

Comment:
@sachinthegreen Is it okay to change the title of this doc to:
```suggestion
Raw IEEE 802.11 packet reception using Promiscuous mode
```
So that it is similar to: https://developer.nordicsemi.com/nRF_Connect_SDK_dev/doc/PR-13896/nrf/protocols/wifi/sniffer_rx_operation.html